api/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.jboss.weld</groupId>
 		<artifactId>weld-parent</artifactId>
-		<version>31</version>
+		<version>36</version>
```

Comment:
Well yeah. We should discuss it somewhere else then, but I don't consider it so problematic. `weld-parent` is just pom.xml and no impl in fact.